src/components/image/image.tsx

```diff
@@ -112,40 +112,59 @@ export class EuiImage extends Component<EuiImageProps, State> {
     const fullScreenDisplay = (
       <EuiOverlayMask onClick={this.closeFullScreen}>
         <EuiFocusTrap clickOutsideDisables={true}>
-          <button
-            type="button"
-            className="euiImage-isFullScreen"
-            onClick={this.closeFullScreen}
-            onKeyDown={this.onKeyDown}>
-            <figure>
-              <img src={url} className="euiImage-isFullScreen__img" alt={alt} />
-              {optionalCaption}
-
+          <figure
+            className="euiImage euiImage--isFullScreen"
+            role="figure"
+            aria-label={caption}>
+            <button
+              type="button"
+              aria-label={`Close ${alt} image full screen`}
+              className="euiImage__button"
+              onClick={this.closeFullScreen}
+              onKeyDown={this.onKeyDown}>
+              <img
+                src={url}
+                alt={alt}
+                className="euiImage--isFullScreen__img"
+              />
               <EuiIcon
                 type="cross"
                 color={fullScreenIconColorMap[fullScreenIconColor]}
-                className="euiImage-isFullScreen__icon"
+                className="euiImage--isFullScreen__icon"
```

Comment:
Same with this className
```suggestion
                className="euiImage-isFullScreen__icon"
```